File: app/src/main/java/ayds/newyork/songinfo/moredetails/fulllogic/NYTimesAPI.java
package ayds.newyork.songinfo.moredetails.fulllogic;


import retrofit2.Call;
import retrofit2.Retrofit;
import retrofit2.converter.scalars.ScalarsConverterFactory;
import retrofit2.http.GET;
import retrofit2.http.Query;

interface NYTimesAPI {
  Call<String> getArtistInfo(@Query("q") String artist);
}

File: app/src/main/java/ayds/newyork/songinfo/moredetails/fulllogic/model/data/external/info/NYTimesAPI.kt
package ayds.newyork.songinfo.moredetails.fulllogic.model.data.external.info

import retrofit2.Response
import retrofit2.http.GET
import retrofit2.http.Query

internal interface NYTimesAPI {
    @GET("articlesearch.json?api-key=fFnIAXXz8s8aJ4dB8CVOJl0Um2P96Zyx")
    fun getArtistInfo(@Query("q") artist: String): Response<String>
}